text/0004-recoverable-transaction-history.md

```diff
@@ -115,7 +117,7 @@ that went to the change subaddress.
 ## Change subaddress
 
 The "change subaddress" is a new standard subaddress, like the default subaddress. It should be used when sending change outputs.
-It is specified to be subaddress index 1.
+It is specified to be subaddress index `u64::MAX`.
```

Comment:
```suggestion
It is specified to be subaddress index `u64::MAX-1`.
```

@dolanbernard you can propose the change in the comment with the +/- "document" icon

Reply:
nice catch tho